app/controllers/reports/regions_controller.rb

```diff
@@ -100,6 +100,12 @@ def details
         @region.source.blood_pressures.for_recent_bp_log.includes(:patient, :facility)
       )
     end
+
+    @population = @region.estimated_population&.population.to_f
+    @is_population_available_for_all_districts = @region.estimated_population&.is_population_available_for_all_districts
+    if @population
+      @hypertension_patient_coverage = (@chart_data[:patient_breakdown][:total_patients].to_f / @population) * 100
+    end
```

Comment:
This feels like it should all live on EstimatedPopulation -- and we only want to calculate this for state or districts, right?